browsers/safari_ios.json

```diff
@@ -149,7 +149,7 @@
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-15-release-notes",
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "612.1.27"
+          "engine_version": "612.1.29"
```

Comment:
https://github.com/WebKit/WebKit/blob/releases/Apple/Safari-15-iOS-15.0/Source/WebKit/Configurations/Version.xcconfig claims this change is wrong.

Reply:
Thanks for confirming!